Documentation/git-commit.txt

```diff
@@ -141,7 +141,9 @@ OPTIONS
 	the commit author is then copied from the first such commit found.
 
 --date=<date>::
-	Override the author date used in the commit.
+	Override the author date used in the commit. Special
+	strings such as `now`, `yesterday`, `noon`, `tea` can be
+	passed as value for the date.
```

Comment:
I wonder whether it would make sense to add an exhaustive list (with possible clarifications e.g. for `tea`) to `Documentation/date-formats.txt`, and then reference the "DATE FORMATS" section in this hunk?

For extra brownie points, this reference could be in the form of `See <<DATE-FORMATS>>` and add a `[[DATE-FORMATS]]` before the first line of `Documentation/date-formats.txt`, for extra markup goodness.

Reply:
Is there an extensive list available anywhere? 

> Please note that there is a strong preference to continue in lower-case after the colon in the first line of the commit message: "Documentation: update [...]" instead of "Documentation: Update [...]".

Will change it

Reply:
Only in the source code, I would think.

*clicketyclick*

https://github.com/git/git/blob/v2.23.0/date.c#L1099-L1112